larq/optimizers_v2.py

```diff
@@ -97,27 +216,21 @@ def _resource_apply_dense(self, grad, var):
         var_t = lq.math.sign(-tf.sign(var * m_t - threshold) * var)
         return tf.compat.v1.assign(var, var_t, use_locking=self._use_locking).op
 
-    @staticmethod
-    def is_binary(var):
-        return "/kernel" in var.name and "quant_" in var.name
+    def _resource_apply_sparse(self, grad, var, indices):
+        raise NotImplementedError()
 
     def get_config(self):
-        fp_optimizer_config = self.fp_optimizer.get_config()
         config = {
             "threshold": self._serialize_hyperparameter("threshold"),
             "gamma": self._serialize_hyperparameter("gamma"),
-            "fp_optimizer": {
-                "class_name": fp_optimizer_config["name"],
-                "config": fp_optimizer_config,
-            },
         }
         return {**super().get_config(), **config}
 
-    @classmethod
-    def from_config(cls, config, custom_objects=None):
-        new_config = deepcopy(config)
-        fp_optimizer = tf.keras.optimizers.deserialize(
-            new_config["fp_optimizer"], custom_objects=custom_objects
-        )
-        new_config.pop("fp_optimizer", None)
-        return cls(fp_optimizer, **new_config)
+    @staticmethod
+    def is_binary_variable(var):
```

Comment:
Maybe add example of usage or a "See also: `CaseOptimizer`"?

Reply:
Added in 35b3d36.